src/aiida/cmdline/params/options/main.py

```diff
@@ -781,3 +782,11 @@ def set_log_level(ctx, _param, value):
     help='Sort the keys of the output YAML.',
     show_default=True,
 )
+
+INCREMENTAL = OverridableOption(
+    '--incremental/--non-incremental',
+    is_flag=True,
+    default=True,
+    show_default=True,
+    help="Incremental dumping of data to disk. Doesn't require using overwrite to clean previous directories.",
+)
```

Comment:
I wonder what is the ratio putting this into the main while `dump-unsealed` not. Both are only used for dumping, right?

Reply:
Longer term, `incremental` will definitely be used again for dumping/mirroring of profile data, while `dump-unsealed` is for now only being used for the process dump. That's why I just kept it there, rather than putting it into `main.py`.